specification/mediaservices/resource-manager/Microsoft.Media/Accounts/stable/2023-01-01/Accounts.json

```diff
@@ -509,6 +509,39 @@
           "description": "The Private Endpoint Connections created for the Media Service account.",
           "type": "array",
           "readOnly": true
+        },
+        "minimumTlsVersion": {
+          "type": "string",
+          "enum": [
+            "Tls10",
+            "Tls11",
+            "Tls12",
+            "Tls13"
+          ],
+          "x-ms-enum": {
+            "name": "MinimumTlsVersion",
+            "values": [
+              {
+                "value": "Tls10",
+                "description": "Minimum TLS version is TLS 1.0."
+              },
+              {
+                "value": "Tls11",
+                "description": "Minimum TLS version is TLS 1.1."
+              },
+              {
+                "value": "Tls12",
+                "description": "Minimum TLS version is TLS 1.2."
+              },
+              {
+                "value": "Tls13",
+                "description": "Minimum TLS version is TLS 1.3."
+              }
+            ],
+            "modelAsString": true
+          },
+          "description": "The minimum TLS version allowed for this account's requests.",
+          "x-nullable": true
```

Comment:
>nullable [](http://example.com/codeflow?start=13&length=8)

x-nullable allows people to explicitly pass 'null' as a value - is that what you want? What do you want that to mean?

Reply:
So what will it mean when someone sends explicit null? That they want you to choose the min version for them?